diff --git a/configure b/configure
index 370b18c..50829de 100755
--- a/configure
+++ b/configure
@@ -309,14 +309,6 @@ if [ "$use_unity" = "Y" ]; then
 fi
 
 LIB='lib'
-if [ -d /usr/lib64 -a -d /lib64 ]; then
-LIB='lib64'
-fi
-DEBIAN_LD=`ld -v 2>&1 | grep -i debian`
-UBUNTU_LD=`ld -v 2>&1 | grep -i ubuntu`
-if [ "$DEBIAN_LD" != "" -o "$UBUNTU_LD" != "" ]; then
-  LIB='lib'
-fi
 
 # --- QT3 START ---
 QT_IM='N'
